src/main/java/org/cryptomator/ui/preferences/UpdatesPreferencesController.java

```diff
@@ -70,18 +120,73 @@ public ReadOnlyStringProperty latestVersionProperty() {
 	}
 
 	public String getLatestVersion() {
-		return latestVersion.get();
+		return latestVersion.isNotNull().get() ? latestVersion.get() : "-";
```

Comment:
Instead of creating a temporary binding, which get gc'd directly afterwards, you can define in the constructor the property differently with `ObservableUtil.mapWithDefault()` method.